web/src/components/Breadcrumbs.tsx

```diff
@@ -1,40 +1,60 @@
 import HomeOutlinedIcon from '@mui/icons-material/HomeOutlined'
-import { styled } from '@mui/material/styles'
+import Box from '@mui/material/Box'
+import MuiBreadcrumbs from '@mui/material/Breadcrumbs'
+import { styled, useTheme } from '@mui/material/styles'
+import useMediaQuery from '@mui/material/useMediaQuery'
 import React, { ReactElement } from 'react'
 import { Link } from 'react-router-dom'
 
-import { UiDirectionType } from 'translations'
-
 import BreadcrumbModel from '../models/BreadcrumbModel'
 import Breadcrumb from './Breadcrumb'
 import JsonLdBreadcrumbs from './JsonLdBreadcrumbs'
 import Icon from './base/Icon'
 
-const opposite = (direction: UiDirectionType) => (direction === 'ltr' ? 'rtl' : 'ltr')
-
-const Wrapper = styled('nav')`
+const StyledBox = styled(Box)`
   margin: 10px 0;
-  text-align: start;
-  white-space: nowrap;
   overflow: hidden;
   width: 100%;
-  direction: ${props => opposite(props.theme.contentDirection)};
+  align-items: center;
 `
 
-const OrderedList = styled('ol')`
-  direction: ${props => props.theme.contentDirection};
-  display: flex;
-  white-space: nowrap;
-  overflow: hidden;
-  list-style: none;
-  margin: 0;
-  padding: 0;
+const StyledMuiBreadcrumbs = styled(MuiBreadcrumbs)<{ length: number }>`
+  & ol {
+    flex-wrap: nowrap;
+    overflow: hidden;
+    align-items: flex-start;
+  }
+
+  & li {
+    overflow: hidden;
 
```

Comment:
```suggestion
& a {
          text-decoration: none;
      }
```

Reply:
Please rather use our own custom `Link` component :) It takes care of all the internal/external link stuff and removes link highlighting.